bin.js

```diff
@@ -27,19 +27,34 @@ const yargs = require('yargs')
         describe: 'The output file to create, - for stdout',
         default: '-'
       })
+      .option('quality', {
+        alias: 'q',
+        describe: 'The jpeg compression quality, between 0 and 1',
+        default: '1',
```

Comment:
I am not entirely sure whether yargs typecasts the default value, but it will definitely typecast the parsed values. Since this is a `number`, it would be best that the default also be a number rather than a string.

```suggestion
        default: 1,
```